baserun/instrumentation/anthropic.py

```diff
@@ -0,0 +1,64 @@
+#!/usr/bin/env python
+import logging
+from types import GeneratorType
+from typing import Collection, Any
+
+from anthropic.resources import Completions, AsyncCompletions
+from anthropic.types import Completion
+from opentelemetry.sdk.trace import Span
+
+from baserun.instrumentation.base_instrumentor import BaseInstrumentor
+from baserun.instrumentation.span_attributes import (
+    SpanAttributes,
+    ANTHROPIC_VENDOR_NAME,
+)
+
+logger = logging.getLogger(__name__)
+
+_instruments = ("anthropic >= 0.3.0",)
+__version__ = "0.1.0"
+
+
+class AnthropicInstrumentor(BaseInstrumentor):
+    """An instrumentor for OpenAI's client library."""
+
+    WRAPPED_METHODS = [
+        {
+            "class": Completions,
+            "function": Completions.create,
+            "span_name": "anthropic.completion",
+        },
+        {
+            "class": AsyncCompletions,
+            "function": AsyncCompletions.create,
+            "span_name": "anthropic.completion",
+        },
+    ]
+
+    def instrumentation_dependencies(self) -> Collection[str]:
+        return _instruments
+
+    @staticmethod
+    def set_request_attributes(span: Span, kwargs: dict[str, Any]):
```

Comment:
This is still missing a few anthropic attributes. See the server side:

              model: span.getModel(),
              max_tokens_to_sample: span.getMaxTokens(),
              stop_sequences: span.getStopList(),
              stream: span.getStream(),
              temperature: span.getTemperature(),
              top_p: span.getTopP(),
              // top_k: span.getTopK(),

Reply:
Ooh ok finally found the optional kwargs. Added them